### PR TITLE
Fix typos in tool output.

### PR DESCRIPTION
**Pull request by cbowns**
I found these while working on some tools.

### CODE REVIEW

File: lib/fastlane/actions/xcodebuild.rb [hunk 1]
```diff
@@ -299,7 +299,7 @@ def self.available_options
           ['workspace', 'The workspace to use'],
           ['scheme', 'The scheme to build'],
           ['build_settings', 'Hash of additional build information'],
-          ['output_style', 'Set the output format to one of: :standard (Colored UTF8 output, default), :basic (black & white ASCII outout)'],
+          ['output_style', 'Set the output format to one of: :standard (Colored UTF8 output, default), :basic (black & white ASCII output)']
```
Comment (KrauseFx) [inline lib/fastlane/actions/xcodebuild.rb]: Did you remove the `,` at the end of the line?

**Comment by cbowns** [comment]
Not intentionally, it got dropped in a merge. One sec…

**Comment by cbowns** [comment]
And now I know how to run `fastlane`'s tests! I'm adulting so great now.

**Comment by KrauseFx** [comment]
Looks great, thanks for your contribution @cbowns :+1:

**Comment by KrauseFx** [comment]
@cbowns Did you see the [countdown repo](https://github.com/fastlane/countdown)?

**Comment by cbowns** [comment]
Yep, I've got that installed now!

**Pull request merged**